plugins/python/src/main/kotlin/io/paddle/plugin/python/dependencies/TempVenvManager.kt

```diff
@@ -98,4 +98,18 @@ class TempVenvManager private constructor(val venv: VenvDir, val project: Paddle
     fun getFilesRelatedToPackage(pkg: IResolvedPyPackage): List<File> {
         return InstalledPackageInfoDir.findByNameAndVersion(venv.sitePackages, pkg.name, pkg.version).files
     }
+    private data class PipInstallArgs(val noDeps: Boolean = true, val noCacheDir: Boolean = false) {
```

Comment:
You can possibly generalize it for PipResolve as well, but if you have reasons not to do it, it's OK, I don't insist.